engine/dlib/src/dlib/http_client.cpp

```diff
@@ -522,11 +522,13 @@ namespace dmHttpClient
         // DEF-2889 most webservers have a header length limit of 8096 bytes
         char buf[8096];
         const int bufsize = sizeof(buf);
-        if(dmSnPrintf(buf, bufsize, "%s: %s\r\n", name, value) > bufsize) {
+        int buf_length = snprintf(buf, bufsize, "%s: %s\r\n", name, value);
```

Comment:
No.
We use dmSnPrintf() because it works on all platforms.
Please revert to dmSnPrintf()

Reply:
I've updated the changes to only handle the return value properly.